signinapp/templates/users.html.jinja2

```diff
@@ -45,19 +45,24 @@
                   <a class="btn btn-secondary"
                     href="{{ url_for('admin.edit_user', user_id=user.id)}}"
                     role="button">Edit</a>
+                  {%- if user.role.name == "student" -%}
+                    <a class="btn btn-info"
+                      href="{{ url_for('admin.edit_student_data', user_id=user.id)}}"
+                      action="submit">Edit Student</a>
+                  {%- elif user.role.guardian -%}
+                    <a class="btn btn-info"
+                      href="{{ url_for('admin.edit_guardian_data', user_id=user.id)}}"
+                      action="submit">Edit Guardian</a>
+                  {%- if user.role.name == "guardian_limited" -%}
+                    <a class="btn btn-warning"
+                      href="{{ url_for('admin.user_promote', user_id=user.id)}}"
+                      action="submit">Create User</a>
+                  {%- endif -%}
```

Comment:
We don't show `guardian_limited` in the user table. I figured it would clutter things unnecessarily. I think showing them only in the separate Guardian table is fine. If so, we can remove this chunk